### PR TITLE
Rename topic pt -> papertrail

### PR DESCRIPTION
**Pull request by lmarburger**
```
$ heroku pt --app morning-edition-feed | wc -l
     100

$ heroku papertrail --app morning-edition-feed | wc -l
     100

$ heroku help
...
  papertrail   #  Search and/or tail logs
...
  pt           #  Search and/or tail logs
...
```

@dickeyxxx can you give this a quick once-over before I ship it?

### CODE REVIEW

**Comment by troy** [comment]
+1

File: index.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-  name: "pt",
-  description: "Search and/or tail logs"
-};
+var extend      = require('util')._extend;
+var command     = require('./commands/pt');
+var description = "Search and/or tail logs";
+
+// `pt` topic kept for backwards-compatibility.
+exports.topics = [{ name: "papertrail", description: description },
+                  { name: "pt",         description: description }];
```
Comment (troy) [inline index.js]: Is it worth appending ` (legacy)` to the `pt` description? No strong preference, but I could see pointing folks to the new canonical one.

**Comment by dickeyxxx** [comment]
lgtm

**Comment by lmarburger** [comment]
Thanks! :ship:

**Pull request merged**